package.json

```diff
@@ -27,31 +25,36 @@
   },
   "files": [
     "/dist",
+    "/src",
     "/schemas"
   ],
   "main": "dist/enigma.js",
+  "module": "src/services/qix/index.js",
   "devDependencies": {
     "after-work.js": "^2.0.0",
     "babel-core": "^6.10.4",
     "babel-loader": "^6.2.4",
-    "babel-preset-es2015": "^6.9.0",
+    "babel-plugin-external-helpers": "^6.22.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-es2015-rollup": "^3.0.0",
     "babel-register": "^6.7.2",
     "bluebird": "^3.4.6",
     "chai-subset": "^1.5.0",
-    "concurrently": "^3.1.0",
-    "empty-module": "0.0.2",
     "eslint": "^3.8.0",
     "eslint-config-airbnb": "^12.0.0",
     "eslint-plugin-import": "^1.16.0",
     "extend": "^3.0.0",
```

Comment:
It's a good question with no good (npm) answer I think. If you use the default `main`, you don't need them. If you use `module`, you would. I guess we can move them to dependencies and live with two tiny dependencies that are not going to be used in the second case. I'll update (and fix `module` which should be pointing to `src/index.js`)